src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,70 @@
+package duke;
+
+import java.time.LocalDate;
+
+
+/**
+ * Command for adding tasks into list
+ */
+public class AddCommand extends Command {
+    private final String input;
+    private final ActionType actionType;
+
+    /**
+     * AddCommand constructor
+     *
+     * @param input User input (without type of task written in front)
+     * @param actionType Dictates type of task to be added
+     */
+    public AddCommand(String input, ActionType actionType) {
+        super(false);
+        this.input = input;
+        this.actionType = actionType;
+    }
+
+    /**
+     * Creates new task and adds it into list, then makes Ui print the task info
+     *
+     * @param tasks List of tasks
+     * @param ui User interface to print task
+     * @param storage File storage object
+     * @throws DukeException if exception encountered
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) throws DukeException {
+        Task newTask = null;
+        switch(actionType) {
+        case ADD_TODO:
+            if (input.length() < 6) {
```

Comment:
I think it's better to use a split() function to deal with the input since in this case, you cannot handle the situation when the user types in a message with a lot of white spaces.